pyshacl/cli.py

```diff
@@ -183,16 +185,13 @@ def str_is_true(s_var: str):
 # parser.add_argument('-h', '--help', action="help", help='Show this help text.')
 
 
-def main():
+def main() -> None:
     basename = os.path.basename(sys.argv[0])
     if basename == "__main__.py":
         parser.prog = "python3 -m pyshacl"
     do_server = os.getenv("PYSHACL_HTTP", "")
     do_server = os.getenv("PYSHACL_SERVER", do_server)
-    if do_server:
-        args = {}
-    else:
-        args = parser.parse_args()
```

Comment:
This still needs to skip `parse_args()` if the HTTP or SERVER env variables are set.
This is to allow the user to run PySHACL in server mode, without passing any cmdline arguments.
Normally omitting cmdline arguments will cause `parse_args()` to throw it's missing arguments error.

Reply:
Restored, using an empty `Namespace` object instead of an empty dictionary so `args` has a more consistent type across both these branch sides.